src/stream/src/executor/source/source_executor.rs

```diff
@@ -183,13 +183,7 @@ impl<S: StateStore> SourceExecutor<S> {
     ) -> StreamExecutorResult<()> {
         self.metrics
             .source_split_change_count
-            .with_label_values(
-                &self
-                    .get_metric_labels()
-                    .iter()
-                    .map(AsRef::as_ref)
-                    .collect::<Vec<&str>>(),
-            )
+            .with_guarded_label_values(&self.get_metric_labels().each_ref().map(AsRef::as_ref))
```

Comment:
A remaining temporary `with_guarded_label_values`. Is this expected?

Reply:
Fixed